iconservice/icon_service_cli.py

```diff
@@ -162,28 +163,37 @@ def _start_process(conf: 'IconConfig'):
     Logger.info(f'stop_process_icon_service!', ICON_SERVICE_CLI)
 
 
-def _is_running_icon_service(conf: 'IconConfig') -> bool:
-    return _check_service_running(conf)
-
-
-def _check_service_running(conf: 'IconConfig') -> bool:
-    Logger.info(f'check_serve_icon_service!', ICON_SERVICE_CLI)
+def _check_if_process_running(conf: 'IconConfig') -> bool:
     proc_title = ICON_SERVICE_PROCTITLE_FORMAT.format(**
                                                       {ConfigKey.SCORE_ROOT_PATH: conf[ConfigKey.SCORE_ROOT_PATH],
                                                        ConfigKey.STATE_DB_ROOT_PATH: conf[ConfigKey.STATE_DB_ROOT_PATH],
                                                        ConfigKey.CHANNEL: conf[ConfigKey.CHANNEL],
                                                        ConfigKey.AMQP_KEY: conf[ConfigKey.AMQP_KEY],
                                                        ConfigKey.AMQP_TARGET: conf[ConfigKey.AMQP_TARGET]})
-    return find_procs_by_params(proc_title)
-
+    cmd_lines = _get_process_command_list()
+    if cmd_lines:
+        for cmdline in cmd_lines:
+            if cmdline.startswith(b'icon_service.') and cmdline.decode() == proc_title:
```

Comment:
We already use setProctitle package in our project.
but that package makes proc name add 12 space bytes on OSX's proc

so we should use strip() if it is alright.
I know it makes performance down. but if you want to fix it we have to make new algorithm.

```suggestion
            if cmdline.startswith(b'icon_service.') and cmdline.decode().strip() == proc_title:
```